examples/chrome/cypress/plugins/index.js

```diff
@@ -6,7 +6,7 @@ module.exports = on => {
 
     if (browser.name === 'chrome') {
       // https://www.ghacks.net/2013/10/06/list-useful-google-chrome-command-line-switches/
-      launchOptions.args.push('--window-size=1280,1024')
+      launchOptions.args.push('--window-size=1920,1080')
```

Comment:
I would add a comment explaining that this will set the size of your screenshots / videos, since this is an example used to explain.